src/modules/ui/components/DataCard/DataCard.styles.js

```diff
@@ -0,0 +1,65 @@
+export const card = ({ theme }) => ({
+  display: "flex",
+  alignItems: "stretch",
+  padding: "1rem",
+  borderRadius: theme.borderRadius,
+  border: `2px solid ${theme.colors.white}`,
+  boxShadow: `5px 5px 20px 0px ${theme.colors.secondaryShadow}`,
+  height: "100%",
+
+  "& h3.ant-typography": {
+    fontSize: "1.125rem",
+    marginTop: ".5rem",
+    marginBottom: ".5rem",
+  },
+
+  "& .ant-card-body": {
```

Comment:
This kind of selectors is computation heavy because the browser needs to test the whole subtree to find if there is any `.ant-card-body` in `&`. Because of that I try to always use `>` operator because it restricts the path to be only one (in the best case scenario.) Assuming there are 3 divs between `&` and `.ant-card-body`, you would write f.e. `& > div > div > div > .ant-card-body`, or when `.ant-card-body` is a children of `&`, than `& > .ant-card-body` would be sufficient :) 

You don't have to write the whole styles because of it, I just want you to be aware of the problem :)